numpy/core/tests/test_indexerrors.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
 from numpy.testing import assert_raises
+import pytest
```

Comment:
If you look at existing tests, you will notice they use `assert_raises`, and do not import `pytest1 directly.